debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RecordMakers.java

```diff
@@ -49,12 +49,12 @@
      * 
      * @param schema the schema information about the MySQL server databases; may not be null
      * @param source the connector's source information; may not be null
-     * @param topicSelector the selector for topic names; may not be null
+     * @param ddlChangeTopicName the name of the topic to which DDL changes are written; may not be null
```

Comment:
I think this is actually a good change, as it is indeed a lot simpler. When we started out, we weren't sure what the database history might actually look like. But it's pretty much always going to be a single topic for the MySQL connector. Passing in the name does simplify things, so very nice.